### DISCUSSION TITLE
账号添加其它alist，登录管理前提下，前台没有上传文件选项

### DISCUSSION THREAD

**Discussion by mangic** [AlistGo/alist#934] [Q&A]
![image](https://user-images.githubusercontent.com/33594145/163091427-7ab7e7da-b887-4893-8746-8c23fb987582.png)
账号alist添加其它alist，成功添加并且访问，但是在登了管理前提下前台没有上传文件选项，其它alist运行正常。

**Comment by xhofe** [comment]
提问前看文档。
